### ISSUE TITLE
Create exemplar form: SS-5

### ISSUE THREAD

**Issue by dmethvin-gov** [usds/us-forms-system#251]
See discussion in #204.

Application for a Social Security Card (https://www.ssa.gov/forms/ss-5.pdf)

**Comment by dmethvin-gov** [comment]
Wow, this is a surprisingly complex form. I'll just note several things here that I've puzzled over as I've been building it.

* The order of fields seems strange. Item 2 asks for the applicant's SSN even though they're applying for one. Items 11 through 13 asks if they've ever applied for an SSN before, the name on the card and the date of birth. All three of those should go together it seems and the SSN question is dependent on whether they applied before?
* Items 9 and 10 are inconsistent. It asks for the mother's name at birth but doesn't ask for the birth name of the father. I guess they're driving at maiden name but it's not clear. Seems there are a lot of ambiguous possibilities here like legal name changes of the father, surrogacy, unwed mothers who can't identify the father, etc.
* Item 4 asks for applicant's date of birth. In the instructions it notes that if the applicant is under 18 then the parent's SSNs are required. If that was absolutely true I'd be inclined to make these required fields based on the birth date value but then the instructions say "If you cannot obtain it checked Unknown" and there are the issues mentioned above as well so 🤷‍♂️ .
* There are some choose-one answers where it would be handy to have an easy way to display text or additional questions based on the answer, but it's cumbersome to do so using USFS.
  - For "Has the applicant ever applied for a Social Security Card" they can answer "Yes", "No", or "I don't know". If the answer is "Yes" more questions appear. For the other answers there are no additional questions but I suspect that the SSA may want to distinguish their answer anyway?
  - When the Citizenship answer is "Legal alien NOT allowed to work" or "Other" they need to provide further documentation, and it would be useful to be able to show more instructions under the answer and potentially some sort of checkbox saying "I have attached the documentation".
